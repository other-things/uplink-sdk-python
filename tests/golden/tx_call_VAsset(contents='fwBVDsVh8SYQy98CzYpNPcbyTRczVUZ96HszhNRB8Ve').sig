r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 74391194370808919431863660371581642436978815131175075211922024123287186137241
