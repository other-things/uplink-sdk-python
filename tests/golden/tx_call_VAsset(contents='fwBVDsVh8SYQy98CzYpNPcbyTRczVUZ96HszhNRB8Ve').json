{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                "args": [
                    {
                        "contents": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                        "tag": "VAsset"
                    }
                ],
                "method": "get"
            },
            "tag": "Call"
        },
        "tag": "TxContract"
    },
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE03NDM5MTE5NDM3MDgwODkxOTQzMTg2MzY2MDM3MTU4MTY0MjQzNjk3ODgxNTEzMTE3NTA3NTIxMTkyMjAyNDEyMzI4NzE4NjEzNzI0MQ=="
}